cmake/modules/AddSwift.cmake

```diff
@@ -1833,7 +1840,21 @@ function(_add_swift_executable_single name)
       BINARY_DIR ${SWIFT_RUNTIME_OUTPUT_INTDIR}
       LIBRARY_DIR ${SWIFT_LIBRARY_OUTPUT_INTDIR})
 
+  # On Windows, libraries are prefixed with "lib", so we need to adjust FAT_LIBRARIES for this.
+  if ("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
+    _list_add_string_prefix(
+        "${SWIFTEXE_SINGLE_LINK_FAT_LIBRARIES}"
+        "lib"
+        SWIFTEXE_SINGLE_LINK_FAT_LIBRARIES)
+  endif()
+
   target_link_libraries("${name}" ${SWIFTEXE_SINGLE_LINK_LIBRARIES} ${SWIFTEXE_SINGLE_LINK_FAT_LIBRARIES})
+
+  # On Windows, we need to link rpcrt4.lib for UUIDs and mincore.lib for various file system APIs
+  if ("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
+    target_link_libraries("${name}" "mincore.lib" "rpcrt4.lib")
+  endif()
```

Comment:
What libraries need this additional linkage?  I added support for target specific linking.  Ive never seen this need to be added.  I think that this may be masking something else.